htmy/renderer/default.py

```
@@ -197,7 +197,13 @@ async def run(self) -> str:
             if async_todos:
                 current_async_todos = async_todos
                 self._async_todos = async_todos = deque()
-                await asyncio_gather(*(process_async_node(n, ctx) for n, ctx in current_async_todos))
+
+                try:
+                    async with create_task_group() as tg:
+                        for n, ctx in current_async_todos:
+                            tg.start_soon(process_async_node, n, ctx)
+                except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)